### DISCUSSION TITLE
Adding custom categories to channel with series

### DISCUSSION THREAD

**Discussion by KeystrokeConcepts** [rrirower/Channel-Feed-Manager#106] [Q&A]
How do we change the basic category titles on the channel like we did in the Direct Publisher Feed seutp.

**Comment by rrirower** [comment]
After opening your feed file, look on the right side of your channel contents. You should see "edit category". Click on it.
 
If you don't, that means you defined all of your categories through the developer portal. In that case, you need to add new categories and assign your content to them.

For more info, please refer to the Wiki.

**Comment by KeystrokeConcepts** [comment]
Thank you. We had not been using your Channel Feed Mgr but will take a look at it now. We had been using Visual Studio to make file changes, thanks
